lib/engine/game/g_18_uruguay/step/buy_sell_par_shares.rb

```diff
@@ -9,9 +9,17 @@ module Step
         class BuySellParShares < Engine::Step::BuySellParShares
           def can_buy?(entity, bundle)
             return false if bundle&.corporation == @game.rptla && !@game.phase.status.include?('rptla_available')
+            return true if loan_limit(entity, bundle)
 
             super(entity, bundle)
           end
+
+          def loan_limit(_entity, bundle)
```

Comment:
This method's name suggests that it will return the number of loans a corporation could take. Something like `excess_loans?` or `too_many_loans?` might be clearer.

Reply:
renamed to excess_loans?